packages/orchestration/src/examples/stakeBld.contract.js

```diff
@@ -34,48 +40,86 @@ export const start = async (zcf, privateArgs, baggage) => {
     baggage,
     privateArgs.marshaller,
   );
-  const makeLocalchainAccountKit = prepareLocalchainAccountKit(
+
+  // Mocked until #8879
+  // Would expect this to be instantiated elsewhere, and passed in as a reference
+  const agoricChainInfo = prepareMockChainInfo(zone);
+
+  const timerBrand = await E(privateArgs.timerService).getTimerBrand();
+  const timestampHelper = makeTimestampHelper(
+    privateArgs.timerService,
+    timerBrand,
+  );
+
+  const makeLocalChainAccountKit = prepareLocalChainAccountKit(
     baggage,
     makeRecorderKit,
     zcf,
+    timestampHelper,
```

Comment:
that seems like an odd factoring. Is `prepareLocalChainAccountKit` supposed to work with an arbitrary implementation of this interface? Doesn't seem like it. Passing the `timerService` seems more straightforward.

Pass `timerBrand` too if `await`ing it inside `prepareLocalChainAccountKit` would be a problem.

Reply:
Agree the caller of `prepareLocalChainAccountKit` should only have to know about `timerService`. See [68c8762](https://github.com/Agoric/agoric-sdk/pull/9380/commits/68c8762a44954c641623a4181abc7ec54fc233e2). I also moved `getTimerBrand()` back to the proposal to avoid an await during prepare.